### ISSUE TITLE
Request: Add Animated Swipe-to-Skip Feature

### ISSUE THREAD

**Issue by Garret-Martin** [jmshrv/finamp#469]
I would like to request the addition of an animated swipe to skip feature for my media player on iOS. Swipe works for switching songs, but it would be nice to have an animation in the swipe such as spotify does. 

btw: I appreciate the excellent work being done on this project. The app looks fantastic! Although I am currently unable to contribute due to my limited experience, I am eager to learn and become an active member of this community. If there are any resources or guidance on how to contribute available for newcomers like me, I would greatly appreciate it. Sorry for having to deal with ios compatibility.

Thank you for all for creating this beautiful application.

**Comment by jmshrv** [comment]
This could likely be done as part of #220. I agree that some more animations in the app would be nice, they're just a bit of a pain to implement :upside_down_face: 

As for contributing, just commenting on issues and stuff is enough :). From a technical standpoint, Finamp uses [Flutter](https://flutter.dev/), which isn't too hard to get the hang of.

**Comment by Chaphasilor** [comment]
I've looked into this briefly, and it should be doable by using a [`PageView`](https://api.flutter.dev/flutter/widgets/PageView-class.html) instead of a `Dismissable` for the now playing bar.

**Comment by vijayabhaskar78** [comment]
Hey, can I contribute to this issue

**Comment by Chaphasilor** [comment]
@vijayabhaskar78 you'll have to check if @edeuss since afaik they were planning on working on this :)

**Comment by vijayabhaskar78** [comment]
Ok thankyou then I will explore other issues

**Comment by draw-code-git** [comment]
Is this issue still open?

**Comment by Chaphasilor** [comment]
@draw-code-git yes, there's still no nice animation. It's shouldn't be hard to do, I just haven't found the time to add it yet. Are you interested in tackling this?
